controllers/dspipeline_controller.go

```diff
@@ -52,6 +52,7 @@ type DSPAReconciler struct {
 	Log                     logr.Logger
 	TemplatesPath           string
 	MaxConcurrentReconciles int
+	RequeueTime             time.Duration
```

Comment:
@gmfrasca raised a concern that we may be bloating this struct

Reply:
Done, updated to get the configs from viper.